### ISSUE TITLE
SuperTux Android package/compilation, status of Google Play org.lethargik.supertux2

### ISSUE THREAD

**Issue by qubodup** [SuperTux/supertux#688]
Hello,

SuperTux is available at https://play.google.com/store/apps/details?id=org.lethargik.supertux2&hl=en as a binary package.

A statement on the official download page would be nice. Something like:

"Android: third party packages of SuperTux are available on Google Play. However we do not own or control these. Use them at your own discretion."

or some other explanation as to the relation of the official SuperTux and the Google Play org.lethargik.supertux2 package (and possibly others?).

I learned that its sources are at https://sourceforge.net/projects/libsdl-android/files/apk/SuperTux/ (apk files are also there) and https://github.com/pelya/supertux

Basically: if the dev team trusts the apk/google play, it'd be nice if they linked to them. If the dev team has no particular reason (not contact or no promise that ads/privacy invasion code won't be added to that version) to trust these packages, it would be nice if the team could clarify that there are no official Android packages.

Instructions for building for Android would be a nice bonus.

**Comment by maxteufel** [comment]
> Instructions for building for Android would be a nice bonus.

Since it is not an official port, we are not aware of these details. However, if @pelya (or anyone else) wants to contribute a guide to our wiki (open a PR to SuperTux/wiki), it will probably be merged.

**Comment by pelya** [comment]
I've added some compilation instructions here: https://github.com/pelya/commandergenius/tree/sdl_android/project/jni/application/supertux
